python/py-pytrie/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pytrie
+version             0.3.1
+platforms           darwin
+license             permissive
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         A pure Python implementation of the trie data structure.
+long_description    ${description}
+
+homepage            https://github.com/gsakkis/pytrie/
+master_sites        pypi:P/PyTrie
+distname            PyTrie-${version}
+
+checksums           rmd160  5ab717fc036aee55504f27586b36a1e37aa269d4 \
+                    sha256  3bb7d122a549d8006c9cba2feecac20bceda9f8c0e2a3fa99a5584cfe0dec523 \
+                    size    94678
+
+python.versions     27 38
```

Comment:
Is there some need for the obsolete py27 version? Let's remove that unless there is another port that needs it for some reason.

Reply:
Good point. Fixed